src/storage/src/hummock/compactor.rs

```diff
@@ -771,23 +774,35 @@ impl Compactor {
             // 1024) as f64;     read_statistics.cnt += 1;
             // }
 
-            if can_concat(&level.table_infos.iter().collect_vec()) {
+            if level.level_type == LevelType::Nonoverlapping as i32 {
```

Comment:
Is non-base sublevel considered `NonOverlapping` according to this `level_type` field?

Reply:
Yes. When there are multiple compute node, it means that every checkpoint will generate several files and they must be overlapping.

In the future, we will support one compute node to generate several overlapping files (we call it spill-to-s3) in once checkpoint and all them overlap with each other because we will generate file when the memory usage of shared-buffer reach a limit.